src/NzbDrone.Core/Extras/Subtitles/SubtitleService.cs

```diff
@@ -175,16 +183,18 @@ public override IEnumerable<ExtraFile> ImportFiles(LocalEpisode localEpisode, Ep
                 }
             }
 
-            var subtitleFiles = new List<Tuple<string, Language, string>>();
+            var subtitleFiles = new List<Tuple<string, Language, string, IEnumerable<string>>>();
```

Comment:
I have been pondering about if the tags should be just put in the single string just after they are parsed (delimit them with . for example), but on the other hand having it as enumberable is more flexible.

Any opinion on that?

Reply:
List/enumerable would be nicer to deal with, I'd also get away from the Tuple and move the return value to a class.

Reply:
Changed to class - f2df494f42333ff5d57baf3c5e5d620cf045c3d8
Tags are stored as enumerable and there's a shorthand for getting them as a string.